vipps-ecom-api.md

```diff
@@ -59,8 +60,53 @@ See also: [How it works](vipps-ecom-api-howitworks.md).
   - [Reserve](#reserve)
   - [The Vipps landing page](#the-vipps-landing-page)
   - [Capture](#capture)
-  - [Reserve capture](#reserve-capture)
-  - [Direct capture](#direct-capture)
+  - [Reserve capture](#reserve-capture-1)
+  - [Direct capture](#direct-capture-1)
```

Comment:
wtf

Reply:
Auto generated by Table of contents :S

Reply:
This is duplicated please remove

Reply:
changing

Reply:
Actually not duplicate, distinct sections

Reply:
what about everything added below?

Reply:
Fixed, TOC auto generate shenaningans